examples/camera.py

```diff
@@ -96,23 +96,71 @@ def move(self, axis_x, axis_y):
     def begin(self):
         # Set the current camera offset so you can draw your scene.
         # Translate using the zoom and the offset.
-        pyglet.gl.glTranslatef(-self.offset_x * self._zoom, -self.offset_y * self._zoom, 0)
+        pyglet.gl.glTranslatef(
```

Comment:
We're targeting a 100-column line width for the pyglet codebase, so I think this is better as one line.

Reply:
Ok, I'll change it back